services/coreai/coreai-mcu/core-ai-techradar/infra/dev/01/patch.yaml

```diff
@@ -43,3 +43,4 @@ spec:
       kubernetes_cluster: ${CLUSTER_NAME}
       kubernetes_namespace: ${TEAM_NAMESPACE}
       kubernetes_label_serviceCode: ${SERVICE_CODE}
+    enablePrivateEndpoint: "true"
```

Comment:
format is off? indentation? also does this need to be in scaffolder/api?

Reply:
I'll add it to API templates... I think indentation looks right to me.

Reply:
Should the PE thing be hard coded? or is there any logic in the HELM that makes it dynamic.. or is it hard coded only?

Reply:
@santosh-gh  as discussed, let us set the values of true/false at the environment level and reference the variable instead.

Reply:
The value is set in the environment level and is referred. But instated of true/false I have set enable/disable as yaml/flux does not support true/false and yes/no.